### PR TITLE
feat: Component location card

### PR DESCRIPTION
**Pull request by DiogoSoaress**
Add location card component

## Screenshots
Screenshot 2022-04-27 at 10 28 46

_mobile_
Screenshot 2022-04-27 at 10 28 39

### CODE REVIEW

File: src/components/careers/Locations/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      country: 'Portugal',
+    },
+    {
+      city: 'A Coruña',
+      street: 'Skalitzer Str.85',
+      postcode: '10997',
+      country: 'Spain',
+    },
+  ]
+
```
Comment (katspaugh) [inline src/components/careers/Locations/index.tsx:52]: Please define these constants outside of the component.
Can you find the actual correct addresses somewhere?

Comment (DiogoSoaress) [reply src/components/careers/Locations/index.tsx]: Just pushed that 😅

**Review by usame-algan — APPROVED** [review]
👍

**Comment by katspaugh** [comment]
You merged this but you didn't move the styled constants outside of the Locations component.

**Pull request merged**